Ben987/Bondage-College
# Feat: GGTS related Chinese translation.

### What is this change?

Chinese translation for GGTS game script. And the translation provided in this commit has untranslated parts in order to guide Chinese players to answer GGTS questions using English.

### Testing
Localhost override.

* Run the following:
```
cd JS/Bondage-College/BondageClub
py -m http.server
```

* Verify translation exists.
